temp/harmonic_analysis.py

```diff
@@ -0,0 +1,96 @@
+"""Pseudocode for PyDPF-Post Harmonic Mechanical Simulations APIs."""
+
+import ansys.dpf.post as dpf
+from ansys.dpf.post import examples
+from ansys.dpf.post.selection import Selection
+
+# Provide Enums for available physics_types and analysis_types
+# from ansys.dpf.post import physics_types, analysis_types
+
+
+harmonic_simulation = dpf.load_simulation(examples.complex_rst)
+# harmonic_simulation = dpf.load_simulation(examples.complex_rst,
+#                                           physics_type=physics_types.mechanical,
+#                                           analysis_type=analysis_types.harmonic)
+
+# -----------------------------------------------------------------------------------------
+# Explore the simulation metadata
+
+# Print information about the simulation
+print(harmonic_simulation)
+
+# Print the mesh
+print(harmonic_simulation.mesh)
+
+# Print the list of constructed geometries
+print(harmonic_simulation.geometries)
+
+# Print the list of boundary conditions
+print(harmonic_simulation.boundary_conditions)
+
+# Print the list of available named selections
+named_selections = harmonic_simulation.named_selections
+print(named_selections)
+
+# Print available frequencies, phase angles, and RPMs
+print(harmonic_simulation.time_freq_support)
+print(harmonic_simulation.frequencies)
+print(harmonic_simulation.phase_angles)
```

Comment:
what is phase_angles?

Reply:
@cbellot000 that would be the list of values for the sweeping phase. I think I took that naming from somewhere...